changelogs/fragments/118-add-gtid_port.yml

```
@@ -0,0 +1,2 @@
+bugfixes:
+  - modules/proxysql_backend_servers - Added missing gtid_port to proxysql_backend_servers module (https://github.com/ansible-collections/community.proxysql/pull/118).
```

```suggestion
minor_changes:
  - proxysql_backend_servers - Added missing ``gtid_port`` argument to the module (https://github.com/ansible-collections/community.proxysql/pull/118).
```
@thanos-k @markuman I think it should be of the `minor_changes:` category as it adds a new argument.